tlog-tiles.md

```diff
@@ -154,6 +159,88 @@ compatibility concerns. If they do, they SHOULD also serve entry bundles, and
 MUST serve the differently-encoded entries at a different URL from the specified
 entry bundle URL.
 
+### Pruning
+
+In some log applications, such as [Certificate Transparency][], entries expire
+and are replaced with renewed versions. As this happens, the total size of the
+log grows, even if the unexpired subset remains fixed. To mitigate this, this
+section defines procedures to *prune* log. Pruning makes some prefix of the log
+unavailable, without changing the tree structure.
+
+Logs maintain a *minimum index* value. The minimum index is the index of the
+first log entry that the log publishes. It MUST be less than or equal to the
+tree size of the log's current checkpoint. In an unpruned log, the minimum index
+is zero.
+
+In response to a request for a full tile `<prefix>/tile/<L>/<N>`, or an entry
+bundle `<prefix>/tile/entries/<N>`, the log MAY return a 410 or 404 response if the
+resource's end index, defined above, is below or equal to the minimum index.
```

Comment:
"or equal" is inconsistent with "The minimum index is the index of the first log entry that the log publishes" above.

Reply:
Oh, did I get the boundary condition wrong? I _think_ it's right? start/end are a half-open interval. We're hiding `[0, min_index)` and keeping `[min_index, infinity)`. If I have a tile `[start, end)` and `end = min_index`, it doesn't intersect with the kept range, so it's okay[*] to prune it.

[*] More specifically, if it doesn't intersect, you can prune the *leaves* of the tile but you *may* still need to the root of the tile. However, the root of the tile is one of the leaves of the parent tile, so we can still delete this tile and evaluate the condition at parent to figure out if the parent tile is needed.